docs/releasing.rst

```diff
@@ -0,0 +1,30 @@
+
+Releasing
+=========
+
+Releasing a version of the Anonlink Entity Service
+--------------------------------------------------
+
+We follow gitflow. Each release has a GitHub milestone associated with it which groups all the features and
+bug fixes together.
+
+Multiple docker images are contained within this repository (e.g., ``backend``, ``frontend``, ``benchmark``) which
+are independently versioned. In general a release involves a new version of both the ``backend`` and the ``frontend``.
+This is because the documentation is baked into the frontend so user visible changes to the backend require a new
+frontend.
+
+- Choose a new version using semantic versioning.
+- Create a branch off the latest ``develop`` called ``release-x.y.z``.
+- Update the versions in the code base (e.g., ``backend/entityservice/VERSION``) of any components that have been
+  changed. As above note if the backend version has changed you must release a new frontend too.
```

Comment:
and the frontend if documentation has been updated, which should be all the time :)